### PR TITLE
Add Department MT and RT success rates

### PR DESCRIPTION
**Pull request by sebastiancaraballo**
## What does this PR do?
- Add Merge Time Success Rate and Time to 2nd Review Success Rate to Department section.
- Add tests.

Preview:
Captura de Pantalla 2020-12-09 a la(s) 16 26 07
Captura de Pantalla 2020-12-09 a la(s) 16 26 12

Resolves [#EM-97](https://rootstrap.atlassian.net/browse/EM-97?atlOrigin=eyJpIjoiZTg1YWQ4YzNjNTc5NGE3ZGE4OGIyMzgwMjgyNDhiMjMiLCJwIjoiaiJ9) & [#EM-95](https://rootstrap.atlassian.net/browse/EM-95?atlOrigin=eyJpIjoiNmE5NzgyMzc4OTk2NGM4ZGJiYmFmODBmZjgwYTdiYTAiLCJwIjoiaiJ9)

### CODE REVIEW

File: app/views/development_metrics/departments.erb [hunk 1]
```diff
@@ -43,7 +52,16 @@
             </div>
           </div>
           <div class="metrics shadow-box">
-            <h4 class="p-3">Time to merge</h4>
+            <h4 class="p-3">
+              Time to merge
+              <% if @review_turnaround_success_rate.present? %>
```
Comment (SandroDamilano) [inline app/views/development_metrics/departments.erb:57]: Shouldn't it be `@merge_time_success_rate`?

File: app/services/builders/chartkick/base.rb [hunk 2]
```diff
@@ -23,6 +23,19 @@ def build_distribution_data(entities)
         end
         entities_by_interval.sort_by { |key, _| key.slice(/[0-9]*[+-]/).to_i }
       end
+
+      def build_success_rate(entities)
+        intervals = build_distribution_data(entities)
+        total = intervals.map { |interval| interval[1] }
+                         .reduce(0) { |sum, num| sum + num }
+        return if total.zero?
```
Comment (santib) [inline app/services/builders/chartkick/base.rb:31]: will the rest of the code work if you return `nil` here? or will it fail with a nil exception when trying to access the data?

Comment (sebastiancaraballo) [reply app/services/builders/chartkick/base.rb:31]: When it returns `nil`, then it doesn't show the rate on the web. Take a look at `app/views/development_metrics/departments.erb`.

File: app/services/builders/chartkick/base.rb [hunk 3]
```diff
@@ -23,6 +23,19 @@ def build_distribution_data(entities)
...
+
+        successful = intervals
+                     .select { |tuple| %w[1-12 12-24].include?(tuple[0]) }
```
Comment (santib) [inline app/services/builders/chartkick/base.rb:34]: `%w[1-12 12-24]` could be a constant at least? This will be configurable in the future too, right?

**Review by santib — APPROVED** [review]

File: app/services/builders/chartkick/base.rb (hunk 3 above)
Comment (sebastiancaraballo) [reply app/services/builders/chartkick/base.rb:34]: That's coming on next PR 😀

**Review by horacio — APPROVED** [review]

**Pull request merged**